protostar/configuration_file/configuration_file.py

```diff
@@ -86,6 +88,32 @@ def set_command_names_provider(
     ):
         pass
 
+    @classmethod
+    def create_appending_cairo_path_suggestion(
+        cls, logger, project_root_path, libs_path, package_name
+    ):
+        protostar_toml = "protostar.toml"
+        shared_command_configs_key = "protostar.shared_command_configs"
+
+        data = tomli.loads(
+            (project_root_path / protostar_toml).read_text(encoding="utf-8")
+        )
+        shared_command_configs = data.get(shared_command_configs_key)
+        if not shared_command_configs:
+            shared_command_configs = {}
+        shared_command_configs["cairo_path"] = (
+            ["..."] if shared_command_configs.get("cairo_path") else []
+        )
+        shared_command_configs["cairo_path"].append(f"{libs_path}/{package_name}/src")
+
+        logger.info(
+            "You may want to add your new library's path to the 'cairo_path' section of your "
+            f"`{protostar_toml}` file"
+        )
+        logger.info(
+            f"\n[{shared_command_configs_key}]\n{tomli_w.dumps(shared_command_configs)}"
+        )
```

Comment:
[Complexity/Functionality] Let's make it simpler and generic. Current suggestion could be invalid if the library doesn't have src directory.

For example:
> You may want to update the cairo-path configuration in the configuration file. 
https://docs.swmansion.com/protostar/docs/tutorials/dependencies-management#adding-a-dependency

Reply:
I think it's good to provide a small example, not only a link to the docs, if you think otherwise, let me know.

Reply:
[make the message more generic](https://github.com/software-mansion/protostar/pull/981/commits/bcd081a9ebfdd25525c4f73224afbdbd1ae2366c)

Reply:
> I think it's good to provide a small example, not only a link to the docs, if you think otherwise, let me know.

A message with an example could be a bit too intrusive (a user needs to learn this once and do not forget about it) and harder to implement (especially because we are migrating the configuration file).